plugins/modules/wafv2_resources_info.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: wafv2_resources_info
+version_added: 1.5.0
+author:
+  - "Markus Bergholz (@markuman)"
+short_description: wafv2_resources_info
+description:
+  - List web acl resources.
+requirements:
+  - boto3
+  - botocore
+options:
+    name:
+      description:
+        - The name wafv2 acl of interest.
+      type: str
+      required: true
+    scope:
+      description:
+        - Scope of wafv2 web acl.
+      required: true
+      choices: ["CLOUDFRONT","REGIONAL"]
+      type: str
+
+extends_documentation_fragment:
+- amazon.aws.aws
+- amazon.aws.ec2
+
+'''
+
+EXAMPLES = '''
+- name: get web acl
+  community.aws.wafv2_resources_info:
+    name: string03
+    scope: REGIONAL
+'''
+
+RETURN = """
+resource_arns:
+  description: Current resources where the wafv2 is applied on
+  sample:
+    - "arn:aws:elasticloadbalancing:eu-central-1:111111111:loadbalancer/app/test03/dd83ea041ba6f933"
+  returned: Always, as long as the wafv2 exists
+  type: list
+"""
+from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule, is_boto3_error_code, get_boto3_client_method_parameters
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import camel_dict_to_snake_dict, ansible_dict_to_boto3_tag_list
+
+try:
+    from botocore.exceptions import ClientError, BotoCoreError
+except ImportError:
+    pass  # caught by AnsibleAWSModule
+
+
+def get_web_acl(wafv2, name, scope, id, fail_json_aws):
+    try:
+        response = wafv2.get_web_acl(
+            Name=name,
+            Scope=scope,
+            Id=id
+        )
+    except (BotoCoreError, ClientError) as e:
+        fail_json_aws(e, msg="Failed to get wafv2 web acl.")
+    return response
+
+
+def list_web_acls(wafv2, scope, fail_json_aws):
+    try:
+        response = wafv2.list_web_acls(
+            Scope=scope,
+            Limit=100
```

Comment:
Won't this module need to support a NextMarker parameter?

Reply:
oh, right!